api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -784,6 +792,12 @@ message HttpConnectionManager {
   // value.
   SetCurrentClientCertDetails set_current_client_cert_details = 17;
 
+  // This field is valid only when :ref:`forward_client_cert_details
+  // <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.forward_client_cert_details>`
+  // is SANITIZE_SET_OR_APPEND_FORWARD and the client connection is mTLS. If the client certificate matches the matcher,
+  // the client certificate information will be appended to the request’s XFCC header. Otherwise, the XFCC header will be sanitized.
+  xds.type.matcher.v3.Matcher append_forward_matcher = 60;
```

Comment:
Sorry, may be I didn't make it clear. But see, the api still a little tricky because the matcher is only a matcher and has no any result. (or the result is only a bool to tell whether the `SANITIZE_SET_OR_APPEND_FORWARD` should be enabled.
So, my question is, why not let the matcher to return a result to tell which policy should be used? SANITIZE_SET or APPEND_FORWARD